src/RadialGauge/RadialGaugeSeries/StackedRadialGaugeSupportingLabel/StackedRadialGaugeSupportingLabel.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import css from './StackedRadialGaugeSupportingLabel.module.css';
+
+export interface StackedRadialGaugeSupportingLabelProps {
+  /**
+   * A label shown at the center
+   */
+  label: string;
+
+  /**
+   * A class name to apply
+   */
+  className?: string;
+
+  /**
+   * A y offset to apply to the label
+   */
+  yOffset?: number;
+}
+
+export const StackedRadialGaugeSupportingLabel: FC<
+  Partial<StackedRadialGaugeSupportingLabelProps>
+> = ({ label, className, yOffset }) => (
+  <>
+    {label && (
+      <text
+        x="0"
+        y={yOffset - 20}
+        textAnchor="middle"
+        alignmentBaseline="middle"
+        className={classNames(className, css.stackedSupportingLabel)}
+      >
+        {label}
+      </text>
+    )}
+  </>
+);
+
+StackedRadialGaugeSupportingLabel.defaultProps = {
+  yOffset: 0
```

Comment:
Optional but It would be nice if we could pass a `position: 'top' | 'bottom'` rather than the `y`. Would make this more complex to calculate though but much better from a consumer perspective.

Reply:
I like the idea in principle, but I think it may make the feature less flexible for other users? By using `top`, and `bottom` I would be defining the spacing between the value text and the descriptive text myself, whereas a `yOffset` value allows the user to define this themselves, so if they have a spacing pattern of `5px` instead of `4px` they can implement it. This flexibility would be taken away by a definition of just `top` or `bottom`.

Reply:
Thats fair - I was just trying to think about the more novice consumer. Lets leave as is then.